app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -157,7 +158,7 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
                     if (pendingAddress != null && pendingAddress.isNotBlank()) {
                         add(HomeModel.PendingAddressChange(pendingAddress))
                     }
-                    if (FeatureFlag.MOVING_FLOW.enabled) {
+                    if (FeatureFlag.MOVING_FLOW.enabled && marketManager.market == Market.SE) {
```

Comment:
Should we move this logic up to the feature flag itself? Now we're spreading it out in various places in the code, right? 🤔

Reply:
I was thinking about that, but the feature flag will be removed when we release moving flow and we want to keep the market logic - so thats why they are separated here.

Reply:
Perhaps we should keep the flag until we reach general availability of the feature? 🙂